specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2019-08-08/acquisitions.json

```diff
@@ -33,31 +33,28 @@
             "$ref": "storage.json#/parameters/SubscriptionIdParameter"
           },
           {
-            "$ref": "storage.json#/parameters/ResourceGroupNameParameter"
-          },
-          {
-            "$ref": "storage.json#/parameters/FarmIdParameter"
+            "$ref": "storage.json#/parameters/LocationParameter"
           },
           {
             "$ref": "storage.json#/parameters/ApiVersionParameter"
-          },
-          {
-            "$ref": "storage.json#/parameters/FilterParameter"
           }
         ],
         "responses": {
           "200": {
             "description": "OK -- The list of acquisitions has been returned.",
             "schema": {
-              "$ref": "acquisitions.json#/definitions/AcquisitionList"
+              "$ref": "#/definitions/AcquisitionList"
             }
           },
-          "404": {
-            "description": "NOT FOUND -- The specified farm cannot be found."
+          "default": {
```

Comment:
I don't think this is required.  I believe the C#/PowerShell automatically generates this class.

Reply:
I change this due to ryansbenson's comment. He says it would be better to leave a default error at this place.